autoconfigure/src/test/java/zipkin2/storage/kafka/ZipkinKafkaStorageAutoConfigurationTest.java

```diff
@@ -216,25 +176,26 @@ public void canOverridesProperty_dependenciesTopicPartitions() {
     context = new AnnotationConfigApplicationContext();
     TestPropertyValues.of(
         "zipkin.storage.type:kafkastore",
-        "zipkin.storage.kafka.dependencies-topic-partitions:2"
+        "zipkin.storage.kafka.dependency-links-topic-partitions:2"
     ).applyTo(context);
     Access.registerKafka(context);
     context.refresh();
 
-    assertThat(context.getBean(KafkaStorage.class).dependenciesTopic.partitions).isEqualTo(2);
+    assertThat(context.getBean(KafkaStorage.class).dependencyLinksTopic.partitions).isEqualTo(2);
   }
 
   @Test
   public void canOverridesProperty_dependenciesTopicReplicationFactor() {
     context = new AnnotationConfigApplicationContext();
     TestPropertyValues.of(
         "zipkin.storage.type:kafkastore",
-        "zipkin.storage.kafka.dependencies-topic-replication-factor:2"
+        "zipkin.storage.kafka.dependency-links-topic-replication-factor:2"
```

Comment:
not super important, but in ES for example, we use the single word 'dependency' in the index name here, and 'span' elsewhere. I think your is more correct, but if you are looking for one less hyphen.. :P